src/server/helpers/auth.js

```diff
@@ -107,15 +107,11 @@ auth.isAuthenticated = (req, res, next) => {
 };
 
 auth.isAuthenticatedForHandler = (req, res, next) => {
-	new Promise((resolve) => {
-		if (req.isAuthenticated()) {
-			resolve();
-		}
+	if (req.isAuthenticated()) {
```

Comment:
Not really async/await related, but I'm mad at myself for how I wrote this before.